actix-web/src/http/header/mod.rs

```diff
@@ -14,6 +14,7 @@ use std::fmt;
 // - the few typed headers from actix-http
 // - header parsing utils
 pub use actix_http::header::*;
+pub use actix_http::Uri;
```

Comment:
`Uri` is already re-exported in the `http` module, i don't think there should be another one here